### ISSUE TITLE
[BUG] [paperless.api] An error occurred checking for available updates

### ISSUE THREAD

**Issue by dz-igner** [paperless-ngx/paperless-ngx#4849]
### Description

Since the update to version 2.0, the entry '[DEBUG] [paperless.api] An error occurred checking for available updates' appears under protocols

### Steps to reproduce

1. Go to protocols
2. Search for line '[DEBUG] [paperless.api] An error occurred checking for available updates'

### Webserver logs

```bash
-
```

### Paperless-ngx version

2.1.0

### Host OS

Debian 12

### Installation method

Docker - official image

### Browser

Firefox, Chrome

**Comment by hermannoffen** [comment]
Probably, this relates to #4773 somehow? I considered it a duplicate at first, but since you are referring to version `2.1.0` as being affected, it seems to be something old/new.

**Comment by shamoon** [comment]
Yes please clarify which version you mean, this should have been fixed on 2.1

```
GET /api/remote_version/
HTTP 200 OK
Allow: GET, HEAD, OPTIONS
Content-Type: application/json
Vary: Accept

{
    "version": "v2.1.0",
    "update_available": false
}
```

**Comment by dz-igner** [comment]
> Yes please clarify which version you mean, this should have been fixed on 2.1
> 
> ```
> GET /api/remote_version/
> HTTP 200 OK
> Allow: GET, HEAD, OPTIONS
> Content-Type: application/json
> Vary: Accept
> 
> {
>     "version": "v2.1.0",
>     "update_available": false
> }
> ```

![grafik](https://github.com/paperless-ngx/paperless-ngx/assets/15924199/28e2b499-3389-4df0-88f4-6aca25497448)

**Comment by shamoon** [comment]
Is your installation blocked from the internet? What happens if you open the api endpoint above?

**Comment by stumpylog** [comment]
It also appears to be fine on my 2.1 installation, no errors
